diff --git a/package.json b/package.json
index d8f9144..429fe9e 100644
--- a/package.json
+++ b/package.json
@@ -40,4 +40,4 @@
   "devDependencies": {
     "sequelize-cli": "^6.6.0"
   }
-}
+}
\ No newline at end of file
diff --git a/public/img/pianco.png b/public/img/pianco.png
new file mode 100644
index 0000000..c417a17
Binary files /dev/null and b/public/img/pianco.png differ
diff --git a/server.js b/server.js
index 89db9a7..4c2280e 100644
--- a/server.js
+++ b/server.js
@@ -1,7 +1,3 @@
-
-require('dotenv').config()
-
-
 const express = require('express')
 const app = express()
 const bodyParser = require('body-parser')
@@ -15,7 +11,7 @@ const passport = require('passport')
 const flash = require('express-flash')
 const session = require('express-session')
 const initializePassport = require('./passport-config')
-
+require('dotenv').config()
 initializePassport(
     passport,
     email => users.findOne({ where: { email: email } }),
@@ -37,7 +33,6 @@ app.use(passport.session())
 // const sequelize = new Sequelize("sqlite::memory:");
 app.use(express.static("public"))
 app.use(bodyParser.json())
-app.use(express.static('public'));
 app.use('/css', express.static(__dirname + 'public/css'))
 
 app.get('/mycart', async (req, res) => {
diff --git a/views/createListing.ejs b/views/createListing.ejs
index c287d63..cc15254 100644
--- a/views/createListing.ejs
+++ b/views/createListing.ejs
@@ -21,7 +21,7 @@
             </div>
             <div class="col-md-6">
                 <label for="price" class="form-label">Price</label>
-                <input type="number" name="price" class="form-control" id="price">
+                <input type="integer" name="price" class="form-control" id="price">
             </div>
             <div class="col-12">
                 <label for="description" class="form-label">Description</label>
diff --git a/views/index.ejs b/views/index.ejs
index a9e19b3..90a0e28 100644
--- a/views/index.ejs
+++ b/views/index.ejs
@@ -17,13 +17,13 @@
     <div class="container justify-content-center mt-5 p-5" style="background-color: #fff9e7;">
         <h3>This is Home page</h3>
         <div class="input-group flex-nowrap justify-content-center" style="width: 30%;">
-            <span class="input-group-text" id="addon-wrapping">Search</span>
             <form action="/" method="post" class="d-flex">
                 <input type="text" name="myCart" class="form-control" aria-label="search"
                     aria-describedby="addon-wrapping">
-                <input type="submit" value="Add" class="btn btn-lg btn-primary">
+                <span class="input-group-text" id="addon-wrapping">Search</span>
             </form>
         </div>
+        <img src="public/img/coffeebeans.jpeg" alt="">
         <div class="container d-flex justify-content-center flex-wrap">
             <% item.forEach(function (product){%>
                 <div class="card m-5" style="width: 18rem;">
